arm/examples/create/create.go

```diff
@@ -1,16 +1,21 @@
-package create
+package main
 
 import (
 	"fmt"
 	"log"
+	"math/rand"
+	"time"
 
 	"github.com/Azure/azure-sdk-for-go/arm/examples/helpers"
 	"github.com/Azure/azure-sdk-for-go/arm/storage"
 	"github.com/Azure/go-autorest/autorest/azure"
 	"github.com/Azure/go-autorest/autorest/to"
 )
 
-func createAccount(resourceGroup, name string) {
+func main() {
+	resourceGroup := "resourceGroupName"
+	name := fmt.Sprintf("gosdktestname%s", randomString(5))
+
 	c, err := helpers.LoadCredentials()
```

Comment:
Agree - I think the examples that are showing the ARM functionality should be transparent and simple and just client id/secret as you suggest. I think we should also make sure we cross-link to the full example

"If you want to see an example of authenticating to Azure via certificate or device authorization, please see..."

I thought it fits in with this PR. If not I can do it later.